### ISSUE TITLE
SIP MESSAGE Valve drop all requests when monitor_interval=-1 and Sip_scanners is empty 

### ISSUE THREAD

**Issue by gvagenas** [RestComm/jain-sip.ext#8]
gov.nist.javax.sip.SIP_MESSAGE_VALVE=org.mobicents.ext.javax.sip.congestion.CongestionControlMessageValve
org.mobicents.ext.javax.sip.congestion.SIP_SCANNERS=
org.mobicents.ext.javax.sip.congestion.CONGESTION_CONTROL_MONITOR_INTERVAL=-1

**Comment by deruelle** [comment]
@gvagenas Did you try to comment the org.mobicents.ext.javax.sip.congestion.SIP_SCANNERS= ?

**Comment by gvagenas** [comment]
@deruelle yes I did and that worked but I think this is a bug because when monitor =-1 and sip scanner is empty,  the valve instead of being disabled it drops all requests.
